src/ThundraWrapper.ts

```diff
@@ -424,6 +504,13 @@ class ThundraWrapper {
         const endTime = Math.min(configEndTime, maxEndTime);
 
         return setTimeout(() => {
+            if (this.debuggerProxy) {
+                // Debugger proxy exists, let it know about the timeout
+                if (!this.debuggerProxy.killed) {
```

Comment:
It might be safer to put process kill in a try/catch and assign `null` to `debuggerProxy` in finally like did at the end of `startDebuggerProxyIfAvailable`